src/zimscraperlib/zim/creator.py

```diff
@@ -146,7 +150,30 @@ def config_indexing(
         self.__indexing_configured = True
         return self
 
+    def _get_illustration_metadata(
+        self,
+        name: str,
+        value: bytes | str | datetime.datetime | datetime.date | Iterable[str],
```

Comment:
If you don't know what to expect, why not declaring `Any`?

Reply:
This is the same type annotation as on validate_metadata; see line 202. To have something narrower, I'd need to split this into a function that checked the name, and perhaps the class of value, and a function that does the real business of extracting the metadata from value.

Reply:
ok